Changes

```diff
@@ -2424,6 +2424,9 @@ OCaml 4.07.0 (10 July 2018)
 - #1779: integrate the Bigarray documentation into the main manual.
   (Perry E. Metzger, review by Florian Angeletti and Xavier Clerc)
 
+- #9398: documented OCAMLRUNPARAM settings
+  (Anukriti Kumar, review by Gabriel Scherer)
+
```

Comment:
This doesn't match up with what's changed - the PR updates the man page for `ocamlrun`'s command line options (this is good, just not what it says here!)